src/app/utils/mockStore.ts

```diff
@@ -2,12 +2,20 @@ import { map } from 'rxjs/operators';
 import { BehaviorSubject } from 'rxjs';
 import { Action } from '@ngrx/store';
 import { getErrors } from '@app/store/selectors/error.selectors';
+import {
+  getTestViewState,
+  getVehicleTestResultModel,
+  selectTestTypeById
+} from '@app/store/selectors/VehicleTestResultModel.selectors';
+import { getPreparers, getTestStations } from '@app/store/selectors/ReferenceData.selectors';
 
-export default class MockStore {
+export class MockStore {
   mockSelector: BehaviorSubject<any>;
+  testTypeId: string;
 
-  constructor(mockSelector) {
+  constructor(mockSelector, testTypeId?: string) {
```

Comment:
please revert

Reply:
You suggested passing a parameter instead of '44'. That's how we can pass it from the unit test, when instantiating the mock selector.